bazel/init/stage_1.bzl

```diff
@@ -287,9 +291,9 @@ filegroup(
     maybe(
         name = "rules_foreign_cc",
         repo_rule = http_archive,
-        sha256 = "4b33d62cf109bcccf286b30ed7121129cc34cf4f4ed9d8a11f38d9108f40ba74",
-        strip_prefix = "rules_foreign_cc-0.11.1",
-        url = "https://github.com/bazelbuild/rules_foreign_cc/releases/download/0.11.1/rules_foreign_cc-0.11.1.tar.gz",
+        sha256 = "9561b3994232ccb033278ade83c2ce48e763e9cae63452cd8fea457bedd87d05",
```

Comment:
what's the need for also bumping rules_foreign_cc as part of this PR?

Reply:
Oh whoops. Let me remove that accidental change.